wcivf/apps/people/tests/test_person_views.py

```diff
@@ -179,13 +179,19 @@ def test_one_candidacy_intro(self):
             f"{self.person.name} is a {person_post.party_name} candidate in {person_post.post.label} constituency in the {election.nice_election_name}.",
         )
 
-    def test_previous_party_affiliations(self):
+    def test_previous_party_affiliations_in_current_elections(self):
```

Comment:
This test is probably trying to do too much. Note to self to refactor into smaller chunks.